scripts/unicode.py

```diff
@@ -20,8 +20,8 @@
 import collections
 import urllib.request
 
-UNICODE_VERSION = "9.0.0"
-UCD_URL = "https://www.unicode.org/Public/%s/ucd/" % UNICODE_VERSION
+UNICODE_VERSION = "13.0.0"
+UCD_URL = "http://www.unicode.org/Public/%s/ucd/" % UNICODE_VERSION
```

Comment:
I changed this to HTTP because https://www.unicode.org/foobar currently has a bad TLS certificate.
This is probably related to this note on their main page:

>APRIL 10, 2020 TECHNICAL ALERT:  the Unicode Consortium’s technical website is hosted in a data center that has experienced a catastrophic failure.  We are working to get back online, but this may take a couple weeks.  We apologize for the inconvenience.

Reply:
Let's keep this as is.

Reply:
@Manishearth 
Can be merged this PR as is?
With future tiny correction.